terraform/monitoring/istio_service_setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from google.cloud import monitoring_v3
+
+def getIstioServiceName(service_name, project_id, zone):
+	return 'ist:' + project_id + '-zone-' + zone + '-cloud-ops-sandbox-default-' + service_name
+
+def findService(client, service_name, project_id, zone):
+	found_service = False
+	full_service_name = getIstioServiceName(service_name, project_id, zone)
+	service = client.service_path(project_id, full_service_name)
+	while not found_service:
+		try:
+			found_service = client.get_service(service)
+		except:
+			found_service = False
+	print("Found " + service_name)
+
+def waitForIstioServices(project_id, zone):
+	client = monitoring_v3.ServiceMonitoringServiceClient()
+	# wait for cart service
+	findService(client, "cartservice", project_id, zone)
```

Comment:
I see only 5 services here. Should we add all of them?

Reply:
Yeah this is true - these are the only 5 services we need for the Terraform monitoring, so it's okay if the others haven't shown up since monitoring provisioning won't fail. Do you think we should try to detect all 10?

Reply:
All 10 are being detected now.